fuel/app/classes/controller/welcome.php

```diff
@@ -29,7 +29,8 @@ class Controller_Welcome extends Controller
 	 */
 	public function action_index()
 	{
-		return Response::forge(View::forge('welcome/index'));
+      $hoge='fugafu';
```

Comment:
__[code_sniffer]__
* Tabs must be used to indent lines; spaces are not allowed
* Expected 1 space before "="; 0 found
* Expected 1 space after "="; 0 found